### DISCUSSION TITLE
Custom templates: add a corresponding entry to a "YAML configuration reloading" panel

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16139] [Configuration]
Custom templates can be reloaded by calling a corresponding service.
It is proposed to add a corresponding entry for this on the "Dev tools -> Yaml" page:
![image](https://user-images.githubusercontent.com/71872483/231196207-37fd0868-54ec-4a27-b85b-a4ab5b16d117.png)

Currently the "YAML configuration reloading" panel is used to list yaml-based integrations.
Although custom templates are not yaml (thank you Petro for a remark), for users it would be more convenient to have a possibility to reload custom templates from the same page.

**Comment by Mariusthvdb** [comment]
as I posted in the community, +1 to this yes, and, maybe even more obvious, as it does not distinguish YAML or not: add it to the quickbar.

these are templates, so Key C: templates would be the obvious thing to do. Yaml or not, They're templates.

**Comment by TheHolyRoger** [comment]
+1 - just rename  it to "Config Reloading" and put all the `reload` services in there
